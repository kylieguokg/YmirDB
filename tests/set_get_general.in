set a b
set a a
set a 2
set b a
get a
get b
set b a a
get b
list entries
BYE
