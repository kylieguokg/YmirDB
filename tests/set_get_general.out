> no such key

> not permitted

> ok

> ok

> [2]

> [a]

> ok

> [a a]

> b [a a]
a [2]

> bye
